notify.js

```diff
@@ -99,6 +108,11 @@
         try {
             new N('');
         } catch (e) {
+            if (SW) {
```

Comment:
This variable is only used once, so could you just do `if (w.navigator.serviceWorker)`?